src/components/ErrorScreen.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from "react";
+import { Box, Button, Center, Flex, Text, Icon } from "@chakra-ui/react";
+import { MdSettings } from "react-icons/md";
+import { useNavigate } from 'react-router-dom';
+
+interface Props {
+  error: Error | undefined;
+}
```

Comment:
Could you use this error message and display it somewhere on the page? Be like "You can provide this error to the HexLabs team"